client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -19,8 +19,14 @@
 	overflow: hidden;
 
 	.domain__primary-flag {
-		vertical-align: middle;
 		margin-left: 8px;
+		background: var( --sidebar-menu-hover-background );
+		color: var( --sidebar-menu-hover-color );
+		font-size: 12px;
```

Comment:
I think this `font-size` is not needed since the `domain-management-list-item__meta` class already defines that (`.domain__primary-flag` is a child of `.domain-management-list-item__meta`).

Reply:
True! I've removed it